Readme.md

```diff
@@ -107,6 +107,14 @@ Android apps are compiled using a build system called Gradle. Dolphin's native c
 however, is compiled using CMake. The Gradle script will attempt to run a CMake build
 automatically while building the Java code.
 
+## Visual Studio "Debug" and "Release" Solution Configurations
```

Comment:
I disagree with including this sort of thing as a fully separate section. It is not our job to explain how IDEs function. Especially considering CMake can generate build files for several of them. At most, referencing that the Release configuration should be used in the Building for Windows section should be good enough.

Reply:
That accomplishes the same goal but with less detail so I'm fine with that. Done.